src/repl-window.ts

```diff
@@ -93,9 +97,13 @@ class REPLWindow {
             })
         })
 
+        paredit.onPareditKeyMapChanged((keymap) => {
+            this.postMessage({ type: "paredit-keymap", keymap: keymap });
+        })
+
         this.panel.onDidDispose((e) => {
             if (this.evaluation)
-                this.evaluation.interrupt();
+                this.evaluation.interrupt().catch(() => {});
```

Comment:
This is a reminder to us, as a team: Lots of the repl window code has these `bare` if statements. When we change stuff in those we should put the braces in, b/c that is part of the, (yet very secret)  code convention we have.

Reply:
Good point. That specific change was to remove the linter warning.